src/web/app/pages-student/student-home-page/student-home-page.component.html

```diff
@@ -1,3 +1,61 @@
-<p>
-  student-home-page works!
-</p>
+<br>
+<br>
+<ng-container *ngFor="let course of courses">
+  <div class="card bg-light top-padded" *ngIf="course.sessions.length">
+    <div class="card-header bg-primary text-white">
+      <strong>[{{ course.id }}]: {{ course.name }}</strong>
+      <span class="float-sm-right">
+        <a routerLink="user={{ user }}&courseid={{ course.id }}" class="btn btn-light btn-sm" type="button">
```

Comment:
Where are you attempting to link here?

Reply:
To the student course details page, which requires details like the course id and the user information...but I'm not too sure yet how the routing works and this doesn't feel correct to me...

Reply:
@joanneong because it is not correct...